build-system/tasks/check-sourcemaps.js

```diff
@@ -115,7 +115,9 @@ function checkSourcemapMappings(sourcemapJson, map) {
 
   // Zeroth sub-array corresponds to ';' and has no mappings.
   // See https://www.npmjs.com/package/sourcemap-codec#usage
-  const firstLineMapping = decode(sourcemapJson.mappings)[1][0];
+  const firstLineMapping = decode(sourcemapJson.mappings)[
+    shouldUseClosure() ? 1 : 0
+  ][0];
```

Comment:
Looks like the comment above is no longer accurate? Could we a) update it to specify how things are different with / without closure, and b) extract the first sub-array index to a separate variable for better readability?